pkg/actor/partitioned_update.go

```diff
@@ -88,40 +95,40 @@ func (up *partitionedUpdate) Act(ctx context.Context, cluster *resource.Cluster)
 	// We need to have it not tell us the version
 	// See https://github.com/cockroachdb/cockroach-operator/issues/200
 
-	dbContainer, err := kube.FindContainer(resource.DbContainerName, &statefulSet.Spec.Template.Spec)
-	if err != nil {
-		return err
+	wantContainer := cluster.GetAnnotationContainerImage()
```

Comment:
`wantContainer` is a poor varaible name.  `containerWanted` - `containerRequired`